src/components/renderLessonsPage/renderLessonsPage.tsx

```diff
@@ -0,0 +1,126 @@
+import "./renderLessonsPage.scss";
+import { 
+  FC, useEffect, useState 
+} from "react";
+import Add from "../icons/add.svg";
+import { BACKEND_URL_LESSONS } from "../../constants";
+import Button from "../Button";
+import Loader from "../Loader";
+import NoLessons from "../body/content/my_studio/img/noLessons.svg"; 
+import { RenderLessonContent } from "./renderLessonsContent";
+import { RenderLessonHead } from "./renderLessonsHead";
+import { useIntl } from "react-intl";
+import { useNavigate } from "react-router-dom";
+
+interface RenderLessonPageProps {
+  classNameWrapper: string;
+  classNameInner: string;
+  isHead: boolean;
+  isRenderLessonHead: boolean;
+  isRenderLessonTitle: boolean;
+  isRenderLessonButton: boolean;
+  isRenderLessonNav: boolean;
+  renderLessonHeadTitle: string;
+  renderLessonHeadStatuses: string[];
+  renderLessonHeadCategories: string[];
+  isRenderLessonContentEdited: boolean;
+  renderLessonContentCategoriesUrl: string;
+  renderLessonContentLessonsUrl: string;
+}
+
+export const RenderLessonPage: FC<RenderLessonPageProps> = (renderProps) => {
+  const intl = useIntl();
+  const navigate = useNavigate();
+  const [isLesson, setIsLesson] = useState(false);
+  const [isNoLesson, setIsNoLesson] = useState(false);
+  const [isLoader, setIsLoader] = useState(true);
+  useEffect(() => {
+    fetch(BACKEND_URL_LESSONS)
+      .then((response) => response.json())
+      .then((lesson) =>
+        lesson !== undefined && lesson.records.length > 0
+          ? (setIsLesson(true), setIsLoader(false))
+          : (lesson.records.length === 0 && setIsNoLesson(true),
+          setIsLoader(false))
+      )
+      .catch((error) => console.log(error));
+  }, []);
+
+  return (
+    <div className={renderProps.classNameWrapper}>
+      <div className={renderProps.classNameInner}>
+        {isLoader ? (
```

Comment:
It is better use && here instead of ?:

isLoader && ...
!isLoader && ...

Reply:
done